File: Scala3/scala3-22311/trigger-program-after-minimization.scala
// FILE: I0.java
public interface I0<T> {
  public default T func(T t) {
    return t;
  }
}
// FILE: I1.java
public interface I1 extends I0<String> {
  @Override
  default String func(String s) {
    return s;
  }
}
// Scala Class
abstract class A0 extends I1 with I0[Object] {
  override def func(s: Object): String = super.func(s)
}

File: Kotlin/KT-74148/trigger-program-after-minimization.kt
// TARGET_BACKEND: JVM
// FILE: A0.java
public abstract class A0<T> {
    public abstract Object func(A0<Object> arg0, T arg1);
}

// FILE: A1.java
public abstract class A1 extends A0<A1> {
    public Object func(A0<A1> arg0, A1 arg1) {
        return null;
    }
}

// FILE: main.kt
class A2: A1()